ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/ToolReporter.scala

```diff
@@ -0,0 +1,157 @@
+// Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.testtool
+
+import java.util.concurrent.atomic.AtomicInteger
+
+import org.apache.commons.lang3.exception.ExceptionUtils
+
+import org.scalatest.{events => e}
+import org.scalatest.events.Event
+import org.scalatest.Reporter
+
+/**
+  * Ledger API Test Tool CLI reporter. Implements scalatest's Reporter interface and prints out colorized reports to the
+  * stdout. Supports very limited set of scalatest events.
+  */
+class ToolReporter extends Reporter {
+
+  final val ansiReset = "\u001b[0m"
+  final val ansiBlue = "\u001b[34m"
+  final val ansiGreen = "\u001b[32m"
+  final val ansiCyan = "\u001b[36m"
+  final val ansiYellow = "\u001b[33m"
+  final val ansiRed = "\u001b[31m"
+
+  private def repeatChar(char: Char, n: Int) = List.fill(n)(char).mkString
```

Comment:
nitpick: this can be really inefficient. Consider creating an array and a String from it via: `String.valueOf`

Reply:
I've switched to `StringLike.*` which is efficient: https://github.com/scala/scala/blob/v2.12.8/src/library/scala/collection/immutable/StringLike.scala#L72